cmd/cli/install.go

```diff
@@ -174,6 +182,19 @@ func (i *installCmd) run(config *helm.Configuration) error {
 		}
 	}
 
+	if strings.EqualFold(i.certificateManager, "cert-manager") {
+		var missingFields []string
+		if i.certmanagerIssuerName == "" {
+			missingFields = append(missingFields, "cert-manager-issuer-name")
+		}
+		if i.caBundleSecretName == "" {
+			missingFields = append(missingFields, "ca-bundle-secret-name")
+		}
+		if len(missingFields) != 0 {
```

Comment:
There are defaults initialized for these as a part of the install flags. It doesn't seem like there will ever be missing fields?